CLIENT-SPECIFICATION.md

```diff
@@ -121,8 +121,8 @@ The structure inside these translation folders is identical to that of the main
 
 Although this specification is about the interface that clients must provide, it is also worth noting that pages are written in standard [CommonMark](https://commonmark.org/), with the exceptions:
 - Non-standard `{{`, `}}`, `{{[` and `]}}` placeholder syntax.
-  - `{{` and `}}` surrounds values in an example that indicates editable values.
-  - `{{[` and `]}}` indicate shortform and longform variants of options which are separated by a single `|`. Shortform on the left, longform on the right.
```

Comment:
These two are subpoints of the original bullet point. To me it makes sense to indent them.

Reply:
To me it it says that each bracket combination is their own exception. As in there are 4 in total.

Reply:
I think this should be indented. Both bullet points are related to the "Non standard ..... placeholder syntax".